src/lib/images/proxy.mjs

```diff
@@ -59,10 +59,19 @@ export const isImageRequest = function (req) {
 
 export const transformImageParams = function (query) {
   const params = {}
-  // eslint-disable-next-line id-length
-  params.w = query.w || query.width || null
-  // eslint-disable-next-line id-length
-  params.h = query.h || query.height || null
+
+  if ((query.w || query.width) && (query.h || query.height)) {
+    const width = query.w || query.width
+    const height = query.h || query.height
```

Comment:
[dust] If you move the assignment here outside of the `if` block you could simplify the check